pkg/didcomm/dispatcher/api.go

```diff
@@ -32,6 +32,23 @@ type DIDCommMsg struct {
 	OutboundDestination *Destination
 }
 
+// MsgType msg type
+type MsgType int
+
+const (
+	// PreState pre state
+	PreState MsgType = iota
+	// PostState post state
+	PostState
+)
+
+// Msg msg
+type Msg struct {
```

Comment:
This looks like a StateMsg from the contents?

Reply:
I change it to StateMsg